panel/tests/widgets/test_tables.py

```diff
@@ -27,6 +24,16 @@
                             reason="Requires latest pandas")
 
 
+def makeMixedDataFrame():
```

Comment:
I used a function here because it was easier to drop in than having to use pytest fixtures.